tsconfig.json

```diff
@@ -25,7 +25,7 @@
     /* Modules */
     "module": "commonjs" /* Specify what module code is generated. */,
     // "rootDir": "./",                                  /* Specify the root folder within your source files. */
-    // "moduleResolution": "node10",                     /* Specify how TypeScript looks up a file from a given module specifier. */
+    "moduleResolution": "Node16",                        /* Specify how TypeScript looks up a file from a given module specifier. */
```

Comment:
I've never seen this before why did you set this?

Reply:
The help resolving modules with multiple entry points, like we have in `@ubiquibot/permit-generation`. Multiple entry points are defined in the package.json to allow code splitting. Without this setting the `include` cannot resolve the path properly.
https://github.com/gentlementlegen/permit-generation/blob/1feff62bc87e406f4d5224b75925f478efa693e1/package.json#L100

https://www.typescriptlang.org/tsconfig#moduleResolution